### PR TITLE
Yard extension bugfix (plus change @throws over to @raise)

### PR DESCRIPTION
**Pull request by ylt**

### CODE REVIEW

File: yard_extensions/define_property_handler.rb [hunk 1]
```diff
@@ -10,36 +10,36 @@ class DefinePropertyHandler < YARD::Handlers::Ruby::Base
...
-          o.signature ||= src
-          register(o)
-          o.docstring = doc if o.docstring.blank?(false)
+    namespace.attributes[scope][name] ||= SymbolHash[:read => nil, :write => nil]
 
-          # Register the object explicitly
-          namespace.attributes[scope][name][type] = o
+    # Show their methods as well
+    {:read => name, :write => "#{name}="}.each do |type, meth|
+      if type == :read ? read : write
```
Comment (henryaddison) [inline yard_extensions/define_property_handler.rb:19]: This line is confusing - too many conditionals in one go

**Review by henryaddison — APPROVED** [review]
Fine but maybe rework the line that's a conditional on the output of a ternary operator because it's hard to read.

File: yard_extensions/define_property_handler.rb (hunk 1 above)
Comment (ylt) [reply yard_extensions/define_property_handler.rb:19]: Yep, it is confusing :)

File overall is based on https://github.com/lsegal/yard/blob/master/lib/yard/handlers/ruby/attribute_handler.rb - I kept things the same where possible.

Comment (henryaddison) [reply yard_extensions/define_property_handler.rb:19]: Ah that makes more sense, I didn't think it was really your style. I leave it to your judgement what you want to do (if anything at all).

**Pull request merged**